static/app/components/profiling/flamegraph/aggregateFlamegraphTreeTable.tsx

```diff
@@ -35,6 +30,227 @@ import {useProfileGroup} from 'sentry/views/profiling/profileGroupProvider';
 
 import {FlamegraphTreeContextMenu} from './flamegraphDrawer/flamegraphTreeContextMenu';
 
+function computeRelativeWeight(base: number, value: number) {
+  // Make sure we dont divide by zero
+  if (!base || !value) {
+    return 0;
+  }
+  return (value / base) * 100;
+}
+
+enum FastFrameCallersTableClassNames {
```

Comment:
how do you feel about using const enums here

Reply:
Good call, let me change it